packages/server/src/users/users.controller.ts

```diff
@@ -144,7 +144,7 @@ export class UsersController {
           // @ts-ignore
           config.admin.chain,
       );
-      user.account = account.identity_id.toLowerCase();
+      user.account = account.identityId.toLowerCase();
```

Comment:
i am pretty sure this will fail when we actually try to call on the node, we need to use identity_id bc swagger generates the wrong param names for the api

Reply:
Ok, I i will ts ignore it then